src/mui/input/EmbeddedArrayInputFormField.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Field } from 'redux-form';
+
+import Labeled from './Labeled';
+import { required } from '../form/validate';
+
+const isRequired = (validate) => {
```

Comment:
since it's the same as in `FormField`, I suggest you extract it from there and reuse it.